server/block/frosted_ice.go

```diff
@@ -0,0 +1,118 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/sound"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// FrostedIce ...
+type FrostedIce struct {
+	solid
+
+	// Age ...
+	Age int
+}
+
+// Instrument ...
+func (FrostedIce) Instrument() sound.Instrument {
+	return sound.Chimes()
+}
+
+// BreakInfo ...
+func (p FrostedIce) BreakInfo() BreakInfo {
+	return newBreakInfo(0.5, alwaysHarvestable, pickaxeEffective, silkTouchOnlyDrop(p))
```

Comment:
According to the wiki, it "can be easily destroyed without [tools](https://minecraft.fandom.com/wiki/Tools)." Please check whether `pickaxeEffective` is applicable here.